### PR TITLE
Fix : Move the labels away from the text fields

### PR DESCRIPTION
**Pull request by Jerome-Herbinet**
The problem :

### CODE REVIEW

**Comment by ChristophWurst** [comment]
Could you also post an *after* screenshot? thanks

**Comment by Jerome-Herbinet** [comment]
> Could you also post an _after_ screenshot? thanks
@ChristophWurst

File: src/components/Properties/PropertyMultipleText.vue [hunk 1]
```diff
@@ -142,6 +142,7 @@ export default {
 	user-select: none;
 	text-align: right;
 	background-size: 16px;
+	padding-right: 7px;
```
Comment (nimishavijay) [inline src/components/Properties/PropertyMultipleText.vue:145]: Good catch! Since this is very close to 8px, we could use the variable `calc(var(--default-grid-baseline) * 2)` which would come up to 8px, what do you think? :)

File: package.json [hunk 2]
```diff
@@ -100,7 +100,8 @@
 		"ts-jest": "^29.0.5",
 		"ts-loader": "^9.4.2",
 		"typescript": "^4.9.4",
-		"vue-template-compiler": "~2.6"
+		"vue-template-compiler": "~2.6",
+		"webpack": "^5.75.0"
```
Comment (ChristophWurst) [inline package.json:104]: unrelated change? webpack should come as peer of ``@nextcloud/webpack-vue-config``

Comment (Jerome-Herbinet) [reply package.json]: solved !

File: src/components/Properties/PropertyMultipleText.vue (hunk 1 above)
Comment (Jerome-Herbinet) [reply src/components/Properties/PropertyMultipleText.vue]: done

File: package.json [hunk 3]
```diff
@@ -64,7 +64,7 @@
 		"qr-image": "^3.2.0",
 		"string-natural-compare": "^3.0.1",
 		"uuid": "^9.0.0",
-		"vue": "~2.6.14",
+		"vue": "^2.7.14",
```
Comment (ChristophWurst) [inline package.json:67]: unrelated?

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
revert all changes in package*

File: package.json (hunk 3 above)
Comment (Jerome-Herbinet) [reply package.json]: revert is done
